class UserNotFoundError(Exception):
    pass

class UniqueUserAlreadyExists(Exception):
    pass
